core/src/main/kotlin/plugability/DokkaContext.kt

```diff
@@ -68,6 +69,49 @@ private class DokkaContextConfigurationImpl(
 
     internal val extensions = mutableMapOf<ExtensionPoint<*>, MutableList<Extension<*>>>()
 
+    private enum class State {
+        UNVISITED,
+        VISITING,
+        VISITED;
+    }
+
+    internal val verticesWithState = mutableMapOf<Extension<*>, State>()
+
+    internal val adjacencyList: MutableMap<Extension<*>, MutableList<Extension<*>>> = mutableMapOf()
+
+    private fun topologicalSort() {
+
```

Comment:
Why so many whitespaces?

Reply:
Hard to tell, was more readable while writing

Reply:
Will change